### DISCUSSION TITLE
Welcome to grafolean Discussions!

### DISCUSSION THREAD

**Discussion by grafolean** [grafolean/grafolean#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dersonmanhique** [comment]
Hello,

How can i change the time zone on the dashboards? It's putting the time in UTC timezone.

**Reply by grafolean** [reply]
Hi @Dersonmanhique, at the moment you can't - sounds like something that should be added though. Would you care to open an issue for it? Should be easy enough to fix this (famous last words... ;) ).

**Reply by grafolean** [reply]
Hello @Dersonmanhique - as of Grafolean v0.4.32 the timezone can be set in user profile:

![image](https://user-images.githubusercontent.com/52440604/130277699-d1cd2299-6311-4f13-a93e-c295d4688212.png)
